### ISSUE TITLE
Can the Caps-Lock key be the shortcut modifier key?

### ISSUE THREAD

**Issue by AielloChan** [gbammc/Thor#28]

**Comment by gbammc** [comment]
I think that is a rare case.

**Comment by AielloChan** [comment]
Yes, but no matter `ctl` `commad` or `option`, there are so many shortcuts bound, and even different applications may have internal bindings.
So I don’t know any good way to solve this kind of problem.🤷‍♂️

**Comment by olavfosse** [comment]
@AielloChan You can [swap caps and ctrl in System Preferences](https://olavfosse.no/Exploring_Emacs_II:_Installation_and_Basic_Usage#Switching_Ctrl_Placement) and bind to ctrl in Thor.

**Comment by immadolf** [comment]
Can I customize the modifier key? Like leader key in vim.

**Comment by vansantcode** [comment]
> Can I customize the modifier key? Like leader key in vim.

Current state, no.
